scripts/upload/create.py

```diff
@@ -17,37 +17,35 @@ def main(overwrite=False, credentials='./credentials.json'):
         overwrite: Bool indicating whether to ask the user before overwriting the db
         db: Name of a local database to write tables to
     """
+    with db_session_with_credentials() as (_engine, session):
+
+        if overwrite:
+            initialize(_engine)
+            LOG.warning('Database cleared!\n')
+            try:
+                with _engine.connect() as connection:
+                    # Autocommit so the user is created before granting access
+                    connection = connection.execution_options(
+                        isolation_level="AUTOCOMMIT")
+                    connection.execute(
+                        sqltext("CREATE USER snow WITH PASSWORD 'hackweek';")
+                    )
+                    connection.execute(
+                        sqltext("GRANT USAGE ON SCHEMA public TO snow;")
+                    )
+            except Exception as e:
+                LOG.error("Failed on user creation")
+                raise e
+
+            for t in ['sites', 'points', 'layers', 'images']:
```

Comment:
These are outdated tables - Are we still using this?

Reply:
> I believe so for any new setup of the database

Then we are missing tables here. This only grants the select privilege to the old schema, and our new one has many more. My vote would be to handle the user creation and granting access permissions in a separate scripts.

@aaarendt - Thoughts?